### PR TITLE
[Android] Support Basic button to check user label features

### PR DESCRIPTION
**Pull request by chulspro**
#### Problem
* Need to verify Basic features on Android chip-tool 
* TE5 supports Read user label/product name and write user label

#### Change overview
* Create new button to support basic fragment and add user label/product name button

#### Testing
* QR Pairing with Wi-Fi
* Update Address to establish CASE session
* Read user label/product name and write user label, then Check attribute value result

### CODE REVIEW

File: src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  override fun onStop() {
+    super.onStop()
+    scope.cancel()
+  }
+
+  private fun updateAddressClick() {
+    try {
+      deviceController.updateDevice(
+              basicClusterFabricIdEd.text.toString().toULong().toLong(),
+              basicClusterDeviceIdEd.text.toString().toULong().toLong()
```
Comment (austinh0) [inline src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt:83]: I think correct indentation here is two spaces past the beginning of `deviceController`?

File: src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+  private fun showMessage(msg: String) {
+    requireActivity().runOnUiThread {
+      basicClusterCommandStatus.text = msg
+    }
+  }
+
+  private suspend fun getBasicClusterForDevice(): BasicCluster {
+    return BasicCluster(
+            ChipClient.getConnectedDevicePointer(requireContext(), basicClusterDeviceIdEd.text.toString().toLong()), 0
```
Comment (austinh0) [inline src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt:139]: Same as above, indentation should be two spaces past the beginning of `return`

**Review by austinh0 — APPROVED** [review]

**Review by Damian-Nordic — APPROVED** [review]

File: src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt (hunk 1 above)
Comment (chulspro) [reply src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt]: Thank you for feedback. fixed to two spaces

File: src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt (hunk 2 above)
Comment (chulspro) [reply src/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/BasicClientFragment.kt]: Thank you for feedback. fixed to two spaces

**Review by msandstedt — APPROVED** [review]

**Pull request merged**